mailers/application_mailer.rb

```diff
@@ -0,0 +1,20 @@
+class ApplicationMailer
+  APPLICATION_EMAIL = 'trainify@gmail.com'.freeze
+  
+  def self.send(*args)
+    new(*args).send
+  end
+
+  def send(params)
+    Thread.new do
+      Pony.mail(
+        to:      params[:email],
+        body:    params[:message],
+        subject: params[:subject],
+        from:    APPLICATION_EMAIL,
+        via:     :sendmail,
+        headers: { 'Content-Type': 'text/html' }
```

Comment:
Think about refactoring using `initialize` + `attr_reader` + extracting static key-value pairs into a separate method

Reply:
@kushniryb But dude, that's already mail wrapper. This method is called from child class via 'super'. In RegistrationMailer there are init, attr_reader and everything. Init in parent class would just break existing working flow. If it looks bad, I'll change it totally. But take a look again please.